projects/1.19/assets/croparia/croparia/lang/zh_cn.json

```
@@ -0,0 +1,455 @@
+{
+  "item.croparia.rf_meter": "RF仪",
+  "item.croparia.flour": "面粉",
+  "item.croparia.obsidian_axe": "黑曜石斧头",
+  "item.croparia.obsidian_pickaxe": "黑曜石镐",
+  "item.croparia.obsidian_shovel": "黑曜石铲",
+  "item.croparia.obsidian_hoe": "黑曜石锄头",
+  "item.croparia.obsidian_sword": "黑曜石剑",
+  "item.croparia.renforced_axe": "钻曜斧",
+  "item.croparia.renforced_pickaxe": "钻曜镐",
+  "item.croparia.renforced_shovel": "钻曜铲",
+  "item.croparia.renforced_hoe": "钻曜锄",
+  "item.croparia.renforced_sword": "钻曜剑",
+  "item.croparia.fire_axe": "火元素斧",
+  "item.croparia.fire_pickaxe": "火元素镐",
+  "item.croparia.fire_shovel": "火元素铲",
+  "item.croparia.fire_hoe": "火元素锄",
+  "item.croparia.fire_sword": "火元素剑",
+  "item.croparia.water_axe": "水元素斧",
+  "item.croparia.water_pickaxe": "水元素镐",
+  "item.croparia.water_shovel": "水元素铲",
+  "item.croparia.water_hoe": "水元素锄",
+  "item.croparia.water_sword": "水元素剑",
+  "item.croparia.air_axe": "风元素斧",
+  "item.croparia.air_pickaxe": "风元素镐",
+  "item.croparia.air_shovel": "风元素铲",
+  "item.croparia.air_hoe": "风元素锄头",
+  "item.croparia.air_sword": "风元素剑",
+  "item.croparia.earth_axe": "土元素斧",
+  "item.croparia.earth_pickaxe": "土元素镐",
+  "item.croparia.earth_shovel": "土元素铲",
+  "item.croparia.earth_hoe": "土元素锄头",
+  "item.croparia.earth_sword": "土元素剑",
+  "item.croparia.wood_multitool": "木制万用工具",

```

```suggestion
  "item.croparia.wood_multitool": "木制多功能工具",
```
继承自库内1.16 1.18的croparia
建议看看库内已有的翻译